src/gfx/process.cpp

```diff
@@ -226,8 +226,8 @@ class Png {
 		if (options.inputSlice.height == 0 && height % 8 != 0) {
 			fatal("Image height (%" PRIu32 " pixels) is not a multiple of 8!", height);
 		}
-		if (options.inputSlice.left + options.inputSlice.width * 8 > width
-		    || options.inputSlice.top + options.inputSlice.height * 8 > height) {
+		if (options.inputSlice.width * 8 > width - options.inputSlice.left
```

Comment:
Wait, we aren't bounds-checking `left` or `top` either, are we?

Reply:
`left` and `top` are unsigned; they can't be OOB by themselves.

Reply:
...wait, this doesn't catch that case though: `./rgbgfx -L 99,99:1,1 test/gfx/oob_slice.png` still segfaults. >_<